src/util/audio.hpp

```diff
@@ -0,0 +1,145 @@
+#pragma once
+#include <Windows.h>
+#include <filesystem>
+#include <mmsystem.h>
+
+#pragma comment(lib, "winmm.lib")
+
+namespace big::audio
+{
+	enum string_code
```

Comment:
According to `CONTRIBUTING.md` you should use UpperCamelCase for enum names
and ALL_UPPER for enum members.